ui/core/components/individual_sim_ui/bulk_tab.tsx

```diff
@@ -398,145 +543,141 @@ export class BulkTab extends SimTab {
 				lastTotal = progressMetrics.totalSims;
 
 				if (!!progressMetrics.finalBulkResult) {
-					// reset state
-					this.pendingDiv.classList.add('hide');
-					this.leftPanel.classList.remove('blurred');
-					this.rightPanel.classList.remove('blurred');
+					this.simUI.rootElem.classList.remove('blurred');
+					this.pendingDiv.remove();
 
 					this.pendingResults.hideAll();
 					bulkSimButton.disabled = false;
 					bulkSimButton.classList.remove('disabled');
 					bulkSimButton.replaceChildren(...defaultState);
+					this.resultsTab.show();
+					window.scrollTo({ top: 0, behavior: 'smooth' });
 				}
 			});
 		});
 
-		const importButton = (
-			<button className="btn btn-secondary w-100 bulk-settings-button">
-				<i className="fa fa-download" /> Import From Bags
-			</button>
-		) as HTMLButtonElement;
-		importButton.addEventListener('click', () => new BulkGearJsonImporter(this.simUI.rootElem, this.simUI, this).open());
-
-		const importFavsButton = (
-			<button className="btn btn-secondary w-100 bulk-settings-button">
-				<i className="fa fa-download" /> Import Favorites
-			</button>
-		);
-		importFavsButton.addEventListener('click', () => {
-			const filters = this.simUI.player.sim.getFilters();
-			const items = filters.favoriteItems.map(itemID => ItemSpec.create({ id: itemID }));
-			this.addItems(items);
+		new BooleanPicker<BulkTab>(booleanSettingsContainer, this, {
```

Comment:
To give more control over where you want to append it, you can pass the container as `null` and then actually:
```
const fastModeCheckbox = new BooleanPicker<BulkTab>(null, this, {})

settingsContainer.appendChild(<>
  {fastModeCheckbox.rootElem}  
  {defaultGemDiv}
</>)
```
etc. That way the order wouldn't matter of how we create all these pickers etc.